source/aggregation-tutorials/multi-field-join.txt

```diff
@@ -4,4 +4,240 @@
 Multi-Field Join
 ================
 
-.. TODO: Create multi-field/one-to-many tutorial
\ No newline at end of file
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: code example, node.js, lookup, aggregation
+
+Introduction
+------------
+
+In this tutorial, you can learn how to use the {+driver-short+} to
+perform a multi-field join. A multi-field join occurs when there are
+multiple fields in the documents of two collections that you use to
```

Comment:
S: I think adding "corresponding" clarifies why/how you can use these multiple fields to match documents together

```suggestion
multiple corresponding fields in the documents of two collections that you use to
```